cli/formatting.go

```diff
@@ -198,16 +234,16 @@ func prettifyAsset(w eos.Asset) string {
 
 }
 
-func prettifyAccounts(accounts []eos.AccountName) string {
+func prettifyAccounts(accounts []eos.AccountName) []string {
 	names := []string{}
 	if len(accounts) == 0 {
-		return "<not voted>"
+		return []string{fmt.Sprintf("%s%s", indentPadding, "<not voted>")}

```

Comment:
@tampajohn it'd be less confusing if instead of `not voted` you displayed something about the proxy, in the scenario where one votes via proxy.

For the list of producers, it'd be better to display with commas in one wrapping line -- people can vote up to 30 BPs so that's starting to wasting a lot of space in the terminal...